src/main/steps/c100-rebuild/respondent-details/address/lookup/AddressLookupPostController.test.ts

```diff
@@ -68,4 +69,36 @@ describe('respondent > address > lookup > AddressLookupPostController', () => {
 
     expect(res.redirect).toHaveBeenCalled();
   });
+
```

Comment:
Please remove this test case from here and append this test case in https://github.com/hmcts/prl-citizen-frontend/pull/585 as the controller has been moved to common place - src/main/steps/c100-rebuild/people/LookupAndManualAddressPostController.ts